### PR TITLE
added compatibility with django 2.X

### PR DESCRIPTION
**Pull request by eathtespagheti**
I've just edited the version check to accept Django versions 2.0 or higher (but not for eventually Django 3.X or higher), everything seems to works fine

### CODE REVIEW

File: drf_generators/management/commands/generate.py [hunk 1]
```diff
@@ -45,15 +45,16 @@ def handle_app_config(self, app_config, **options):
             views = options['views'] if 'views' in options else False
             urls = options['urls'] if 'urls' in options else False
 
-        elif django.VERSION[1] >= 8:
+        elif django.VERSION[1] >= 8 or (django.VERSION[0] == 2
+                                        and django.VERSION[1] >= 0):
```
Comment (Brobin) [inline drf_generators/management/commands/generate.py]: This is failing the style check, `and` operator should be on the previous line

File: drf_generators/management/commands/generate.py [hunk 2]
```diff
@@ -45,15 +45,16 @@ def handle_app_config(self, app_config, **options):
...
             force = options['force']
             format = options['format']
             depth = options['depth']
             serializers = options['serializers']
             views = options['views']
             urls = options['urls']
         else:
-            raise CommandError('You must be using Django 1.7, 1.8 or 1.9')
+            raise CommandError('You must be using Django 1.7, 1.8, 1.9, 2.0+')
```
Comment (Brobin) [inline drf_generators/management/commands/generate.py]: Could you remove 1.7 and 1.8 while we're at it? Also add 1.10. I seem to have overlooked this on the last update.

**Review by Brobin — CHANGES_REQUESTED** [review]
I also need 2.0 added to the build and to the badge on the readme. Thanks for the help!

```yml

env:
  - DJANGO=1.10 DRF=3.5
  - DJANGO=2.0 DRF=3.5
```

File: drf_generators/management/commands/generate.py (hunk 1 above)
Comment (Fabi0Z) [reply drf_generators/management/commands/generate.py]: My linter give me a warn if i put it on the previous line

    W504 line break after binary operator

**Review by eathtespagheti — COMMENTED** [review]
My linter give me a warn if i put the `and` in the previous line in `generate.py`

    W504 line break after binary operator

so I think that my solution is better, correct me if I'm wrong

**Comment by Brobin** [comment]
Huh, conflicting warning messages, that's fun :smile:  I'm okay with your way, just add  `# NOQA` to that line so the build passes.

**Comment by eathtespagheti** [comment]
Done

**Pull request closed without merging**